tools/js-sdk-release-tools/src/mlc/clientGenerator/utils/typeSpecUtils.ts

```diff
@@ -14,8 +17,12 @@ export async function generateTypeScriptCodeFromTypeSpec(
     options: ModularClientPackageOptions,
     originalVersion: string | undefined,
     packageDirectory: string
-): Promise<void> {
+): Promise<void> {    
     const tspConfigPath = join(options.typeSpecDirectory, 'tspconfig.yaml');
+    console.log(`tspConfigPath: ${tspConfigPath}`);
```

Comment:
please also update for RLC

Reply:
you can search for `tsp-client` to find other generation that uses tspconfig.yaml